src/assets/js/fields/section-repeater/sections/nestedSubSection.js

```diff
@@ -84,6 +84,11 @@ Vue.component("nested-sub-section", {
 			const sectionToAdd = section.name || section.label;
 
 			const forbidenSection = this.rows.filter(value => {
+
+				if("Field" === value.name){
```

Comment:
I don't like that this is targeted by name which should be translated btw. I would like to keep this change out of this repository and make it only in the Slack extension.